src/types/sequence.rs

```diff
@@ -289,15 +302,33 @@ where
     Ok(v)
 }
 
+fn get_sequence_abc(py: Python<'_>) -> &PyType {
+    SEQUENCE_ABC
+        .get_or_init(py, || {
+            py.import("collections.abc")
+                .expect("coud not import 'collections.abc'")
```

Comment:
I wonder if it would be nice to avoid the panics by storing `PyResult<Py<PyType>>` in the static? Either that or we could finish off #2398 .

Reply:
This is a good idea. The public API (`register`) already returns a Result so this should be pretty straightforward I think.